composer.json

```diff
@@ -25,14 +25,13 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 || ~8.0.0",
```

Comment:
as 5.x support is dropped, such exception tests are also outdated:
- https://github.com/laminas/laminas-math/blob/3.2.0/src/Rand.php#L90

Reply:
and can port to use 7.1 list destructurizing here:
- https://github.com/laminas/laminas-math/blob/3.2.0/src/Rand.php#L116

perhaps the cleanups should be performed in separate or as they are not required changes.

Reply:
The second change can be already made with current PHP versions, so made a PR:
- https://github.com/laminas/laminas-math/pull/6

Reply:
> as 5.x support is dropped, such exception tests are also outdated:

Deferred for now - let's drop them elsewhere (different patch)

Reply:
I've created PR for this:
- https://github.com/laminas/laminas-math/pull/7